src/components/FeatureViews/Accounts/UserProfile.js

```diff
@@ -2,13 +2,13 @@ import React, { useCallback, useEffect, useMemo, useState } from "react";
 import { Redirect, useParams } from "react-router-dom";
 import { useSelector } from "react-redux";
 
-import Badge from "@material-ui/core/Badge";
 import BioIcon from "@material-ui/icons/PersonOutlined";
 import ContactIcon from "@material-ui/icons/ContactPhoneOutlined";
 import CoursesIcon from "@material-ui/icons/SchoolOutlined";
 import CurrentSessionsIcon from "@material-ui/icons/AssignmentOutlined";
 import Grid from "@material-ui/core/Grid";
 import Hidden from "@material-ui/core/Hidden/Hidden";
+import { LabelBadge } from '../../../theme/ThemedComponents/Badge/LabelBadge';
```

Comment:
I would recommend moving this to the bottom of your import statements to group all the material-ui imports together. Also use double quotes to match the other imports.